src/rotator_library/providers/iflow_provider.py

```
@@ -314,14 +408,71 @@ def _convert_chunk_to_openai(self, chunk: Dict[str, Any], model_id: str):
+            # Standard content chunk - pass through
             yield {
                 "choices": choices,
                 "model": model_id,
                 "object": "chat.completion.chunk",
-                "id": chunk.get("id", f"chatcmpl-iflow-{time.time()}"),
-                "created": chunk.get("created", int(time.time())),
+                "id": chunk_id,
+                "created": chunk_created,
+            }
+
+    def _parse_thinking_content(
```

Ruff (0.15.4)

[warning] Missing return type annotation for private function `_parse_thinking_content`

(ANN202)